src/file/file.rpc.ts

```diff
@@ -222,3 +233,44 @@ class ResFile {
     this.updatedAt = file.updatedAt.getTime();
   }
 }
+
+  // ******************** LOGGING FUNCTIONS ******************** */
+
+/**
+ * logs the activity before anything is done.
+ * @param methodName - name of the method activated.
+ * @param fields - parameters sent in the request.
+ * @param traceID - the current trace's id.
+ */
+function logOnEntry(methodName : string, fields: any, traceID: string) : void {
+  const description = JSON.stringify(fields);
+  log(Severity.INFO, methodName, description, traceID);
```

Comment:
There's no indication in the log that it's a request log, like there's on a response log.

Reply:
done.